en/docs/develop/advanced-development/applying-security-to-an-api.md

```diff
@@ -128,38 +130,41 @@ and restart the Micro Integrator.
 
 Add the handler to the REST API:
 
-```xml
-<api xmlns="http://ws.apache.org/ns/synapse" name="StockQuoteAPI" context="/stockquote">
-       <resource methods="GET" uri-template="/view/{symbol}">
-          <inSequence>
-             <payloadFactory media-type="xml">
-                <format>
-                   <m0:getQuote xmlns:m0="http://services.samples">
-                      <m0:request>
-                         <m0:symbol>$1</m0:symbol>
-                      </m0:request>
-                   </m0:getQuote>
-                </format>
-                <args>
-                   <arg evaluator="xml" expression="get-property('uri.var.symbol')"/>
-                </args>
-             </payloadFactory>
-             <header name="Action" scope="default" value="urn:getQuote"/>
-             <send>
-                <endpoint>
-                   <address uri="http://localhost:9000/services/SimpleStockQuoteService" format="soap11"/>
-                </endpoint>
-             </send>
-          </inSequence>
-          <outSequence>
-             <send/>
-          </outSequence>
-          <faultSequence/>
-       </resource>
-       <handlers>
-         <handler class="org.wso2.rest.BasicAuthHandler"/>
+=== "REST API"
+    ```xml
+    <api xmlns="http://ws.apache.org/ns/synapse" name="StockQuoteAPI" context="/stockquote">
+        <resource methods="GET" uri-template="/view/{symbol}">
+            <inSequence>
+                <payloadFactory media-type="xml">
+                    <format>
+                        <m0:getQuote xmlns:m0="http://services.samples">
+                            <m0:request>
+                                <m0:symbol>$1</m0:symbol>
+                            </m0:request>
+                        </m0:getQuote>
+                    </format>
+                    <args>
+                        <arg evaluator="xml" expression="get-property('uri.var.symbol')"/>
+                    </args>
+                </payloadFactory>
+                <header name="Action" scope="default" value="urn:getQuote"/>
+                <send>
+                    <endpoint key="SimpleStockQuoteEndpoint"/>
+                </send>
```

Comment:
Shouldn't there a respond mediator here? Is this working with this?

Reply:
Let's replace this with a call mediator

Reply:
> Shouldn't there a respond mediator here? Is this working with this?

Yes, it secures the API by requiring user name and password. 
I'll replace this with `call`

Reply:
Addressed with https://github.com/wso2/docs-mi/pull/371/commits/d248c358e25efa7125c72ec949f8b0e66730c6be